CloudAppSecurityDocs/troubleshooting-proxy-url.md

```diff
@@ -40,10 +40,12 @@ So even though Fabrikam does not actually use Defender for Cloud Apps, they see
 
 ## Here's why you see `*.mcas.ms`, `*.mcas-gov.us`, or `*.mcas-gov.ms` in your URL
 
-First of all, you have not been phished! This kind of URL is expected and indicates that your organization applies additional security controls to protect business-critical data.
+This kind of URL is expected and indicates that your organization applies additional security controls to protect business-critical data.
 
 They do this by using Defender for Cloud Apps, a solution for protecting your organization's cloud environment, to replace all relevant URLs and cookies relating to cloud apps that you use.
 
 So when you try accessing a cloud app such as Salesforce, SharePoint Online, or AWS, you will notice that its URL is suffixed with `.mcas.ms`, `.mcas-gov.us`, or `.mcas-gov.ms`. For example, when using the XYZ app, the URL you're used to seeing changes from `XYZ.com` to `XYZ.com.mcas.ms`.
 
 If the URL does not exactly match one of the replacement patterns (for example, `<app_site>.com` is not replaced with `<app_site>.com.mcas.ms`), or if you have additional concerns, contact your IT department.
+
+In case you do not recognize the remaining portion of the URL (e.g.: **myurl.com**.mcas.ms) as associated with any of your business apps, it is recommended that you investigate the issue further and consider blocking the URL to avoid any potential security risks.
```

Comment:
```suggestion
If you don't recognize the remaining portion of the URL (for example, **myurl.com**.mcas.ms) as associated with any of your business apps, it's recommended that you investigate the issue further and consider blocking the URL to avoid any potential security risks.
```
Minor edits for Microsoft style